cdc/sink/dmlsink/txn/worker.go

```diff
@@ -122,27 +117,39 @@ func (w *worker) runLoop() error {
 				zap.Int("workerID", w.ID))
 			return nil
 		case txn := <-w.txnCh.Out():
+			// we get the data from txnCh.out until no more data here or reached the state that can be flushed.
+			// If no more daa in txnCh.out, and also not reached the state that can be flushed,
+			// we will wait for 10ms and then do flush to avoid too much flush wish small amount of txns.
 			if txn.txnEvent != nil {
 				needFlush = w.onEvent(txn)
+				if !needFlush {
+					delay := time.After(w.flushInterval)
```

Comment:
IMO, it is better to new a `ticker` outside the `for loop` and reuse it by ticker.Reset() instead of to new a timer every time.

Reply:
ticker.Reset() can't clear the value in channel, if there is a value in ticker.C. So I think ticker.Reset() here can not be useful.